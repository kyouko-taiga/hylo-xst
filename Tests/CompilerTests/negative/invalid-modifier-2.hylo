//! no-std

// Invalid access and indirect modifiers are diagnose after parsing.

trait P {
  public fun f()
}
