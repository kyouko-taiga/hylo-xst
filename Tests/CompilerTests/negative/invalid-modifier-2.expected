negative/invalid-modifier-2.hylo:6.3-9: error: cannot apply access modifiers on trait requirements
  public fun f()
  ~~~~~~
